ampersand-collection-view.js

```diff
@@ -25,23 +29,24 @@ function CollectionView(spec) {
     this.listenTo(this.collection, 'refresh reset', this._reset);
 }
 
-_.extend(CollectionView.prototype, BBEvents, {
+assign(CollectionView.prototype, BBEvents, {
     // for view contract compliance
     render: function () {
         this._renderAll();
         return this;
     },
     remove: function () {
-        _.invoke(this.views, 'remove');
+        invoke(this.views, 'remove');
         this.stopListening();
     },
     _getViewByModel: function (model) {
-        return _.find(this.views, function (view) {
+        return find(this.views, function (view) {
             return model === view.model;
         });
     },
     _createViewForModel: function (model, renderOpts) {
-        var view = new this.view(_({model: model, collection: this.collection}).extend(this.viewOptions));
+	    var defaultViewOptions = {model: model, collection: this.collection};
```

Comment:
Seems we've gained a couple of extra indent spaces here? At least according to the github ui.